dr_ensenso_node/src/ensenso.cpp

```diff
@@ -483,6 +487,64 @@ class EnsensoNode: public ros::NodeHandle {
 		return true;
 	}
 
+	bool onGetCameraParams(dr_ensenso_msgs::GetCameraParams::Request & req, dr_ensenso_msgs::GetCameraParams::Response & res) {
+		if (true
+			&& (req.camera) == dr_ensenso_msgs::GetCameraParams::Request::MONO
+			&& (req.camera) == dr_ensenso_msgs::GetCameraParams::Request::LEFT
+			&& (req.camera) == dr_ensenso_msgs::GetCameraParams::Request::RIGHT
```

Comment:
Shouldn't this be `req.camera != ....` ?

Reply:
Hmm indeed, that is was it used to be. Strange.